src/robot/libraries/dialogs_ipy.py

```diff
@@ -12,32 +12,183 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+import wpf
```

Comment:
This module doesn't seem to be used. Is the import required to initialize something on .NET or could it be removed? If it is required, adding a comment briefly explaining it would be nice.